### DISCUSSION TITLE
Issues with xscreensaver

### DISCUSSION THREAD

**Discussion by dabeegmon** [lxqt/lxqt#2270] [General]
Greetings 

Dunno where to ask but - - - - 
(just in case there is some bearing from the hardware - - - - AMD 3800X, Radeon 570X, 4 - 1920x1080 + 1 - 4k monitors, on Devuan Daedalus and a system loaded for bear! (grin!))
After the last system reboot (adding raid array and hdds from previous system means a restart after the work is done) the screensaver is working great. Things moved to the various patterns and then to screen blanking and shut off as set up. 
I run lots of workspaces (virtual desktops). 
That last reboot was yesterday late afternoon. 
This morning everything is still working well (AFAICT) but then I added another browser back into the mix (Brave) and then when I'm coming back inside I notice that the screensaver isn't working. 
Checked and the daemon is listed as needing a restart. 

Any ideas to what's causing the xscreensaver daemon to die?

Ways to fix? 

TIA

**Comment by darkshram** [comment]
[Xscreensaver](https://www.jwz.org/xscreensaver/download.html) is not a LXQt component (it may be required by some distro packages, but is not maintained by LXQt developers). Since you are using Devuan, you should fill a bug report at [Devuan bug tracker](https://bugs.devuan.org/) because it seems to be a Devuan specific issue.

**Reply by tsujan** [reply]
> it's required,

A screensaver isn't required by LXQt itself (I don't have any) but may be installed by distro meta packages for LXQt.
